src/items/itemsTable.component.tsx

```diff
@@ -0,0 +1,282 @@
+import React from 'react';
+import { Box, Button, Tooltip, Typography } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import ItemDialog from './itemDialog.component';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import ClearIcon from '@mui/icons-material/Clear';
+import {
+  PropertyFiltersType,
+  findPropertyValue,
+} from '../catalogue/items/catalogueItemsTable.component';
+import {
+  MRT_Row,
+  MaterialReactTable,
+  useMaterialReactTable,
+  type MRT_ColumnDef,
+  type MRT_ColumnFiltersState,
+} from 'material-react-table';
+import {
+  CatalogueCategory,
+  CatalogueItem,
+  Item,
+  UsageStatusType,
+} from '../app.types';
+import { MRT_Localization_EN } from 'material-react-table/locales/en';
+import { useItems } from '../api/item';
+
+export interface ItemTableProps {
+  catalogueCategory: CatalogueCategory;
+  catalogueItem: CatalogueItem;
+}
+
+export function ItemsTable(props: ItemTableProps) {
+  const { catalogueCategory, catalogueItem } = props;
+
+  const noResultsTxt =
+    'No results found: Try adding an item by using the Add Item button on the top left of your screen';
+  const { data, isLoading } = useItems(undefined, catalogueItem.id);
+  // SG header + SG footer + tabs #add breadcrumbs + Mui table V2
+  const tableHeight = `calc(100vh - (64px + 36px + 50px + 125px))`;
```

Comment:
This should use `getPageHeightCalc` from `utils.tsx` in #125.

Reply:
That hasn't been merged into develop yet